apps/web/components/getting-started/components/ConnectedCalendarItem.tsx

```diff
@@ -14,7 +14,7 @@ interface IConnectedCalendarItem {
     userId?: number | undefined;
     integration?: string | undefined;
     externalId: string;
-    domainWideDelegationCredentialId?: string | null;
```

Comment:
Ensure that `domainWideDelegationCredentialId` has to be passed and not accidentally missed